workshops/views.py

```diff
@@ -1349,10 +1349,7 @@ def event_import(request):
     This is used to read metadata from workshop website and then fill up fields
     on event_create form."""
 
-    # TODO: remove POST support completely
-    url = request.POST.get('url', '').strip()
-    if not url:
-        url = request.GET.get('url', '').strip()
+    url = request.GET.get('url', '').strip()
```

Comment:
`update_from_url.js` used a POST request and `event_import` view had support for both.
If I had to add the functionality of `update_from_url.js` to `ConferenceImport` view, I would have to add support for POST requests too.
Simply removing POST support was the easier way out, since it didn't break anything and was also on the deprecation list.